libs/components/modals/src/lib/modules/modal/modal-footer.component.scss

```diff
@@ -16,6 +16,16 @@
   .sky-btn + .sky-btn-link {
     margin-left: -2px;
   }
+
+  .sky-modal-footer-errors {
+    max-height: 100px;
+    overflow-y: auto;
+    margin-bottom: var(--sky-margin-stacked-lg);
```

Comment:
Instead of adding the margin here, can you remove the rule below and let the last status indicator serve as the bottom margin? Then you can use the `sky-margin-stacked-lg` CSS class on `sky-status-indicator`.

```
.sky-modal-footer-errors {
  max-height: 100px;
  overflow-y: auto;
}
```

In the template: 

```
<sky-status-indicator class="sky-margin-stacked-lg">
```

Reply:
updated